cassandra-quarkus 1.1.3 not compatible with Quarkus 3

Blocked by https://github.com/datastax/cassandra-quarkus/issues/221

This is the error happening in [camel-4](https://github.com/apache/camel-quarkus/pull/4479) branch: 

```
[ERROR] cqlStatementViaHeader{boolean}[1]  Time elapsed: 0.011 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class com.datastax.oss.quarkus.deployment.internal.CassandraClientProcessor
        at io.quarkus.test.junit.QuarkusTestExtension.throwBootFailureException(QuarkusTestExtension.java:625)
        at io.quarkus.test.junit.QuarkusTestExtension.interceptTestClassConstructor(QuarkusTestExtension.java:696)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.api.extension.InvocationInterceptor.interceptTestClassConstructor(InvocationInterceptor.java:73)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:62)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestClassConstructor(ClassBasedTestDescriptor.java:363)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateTestClass(ClassBasedTestDescriptor.java:310)
        at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.instantiateTestClass(ClassTestDescriptor.java:79)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateAndPostProcessTestInstance(ClassBasedTestDescriptor.java:286)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$4(ClassBasedTestDescriptor.java:278)
        at java.base/java.util.Optional.orElseGet(Optional.java:364)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$5(ClassBasedTestDescriptor.java:277)
        at org.junit.jupiter.engine.execution.TestInstancesProvider.getTestInstances(TestInstancesProvider.java:31)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$prepare$0(TestMethodTestDescriptor.java:105)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:104)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:68)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$prepare$2(NodeTestTask.java:123)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.prepare(NodeTestTask.java:123)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:90)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask$DefaultDynamicTestExecutor.execute(NodeTestTask.java:226)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask$DefaultDynamicTestExecutor.execute(NodeTestTask.java:204)
        at org.junit.jupiter.engine.descriptor.TestTemplateTestDescriptor.execute(TestTemplateTestDescriptor.java:142)
        at org.junit.jupiter.engine.descriptor.TestTemplateTestDescriptor.lambda$execute$2(TestTemplateTestDescriptor.java:110)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1006)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
        at org.junit.jupiter.engine.descriptor.TestTemplateTestDescriptor.execute(TestTemplateTestDescriptor.java:110)
        at org.junit.jupiter.engine.descriptor.TestTemplateTestDescriptor.execute(TestTemplateTestDescriptor.java:44)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.RuntimeException: Failed to load steps from class com.datastax.oss.quarkus.deployment.internal.CassandraClientProcessor
        at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:219)
        at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:107)
        at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:331)
        at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
        at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:60)
        at io.quarkus.test.junit.QuarkusTestExtension.doJavaStart(QuarkusTestExtension.java:222)
        at io.quarkus.test.junit.QuarkusTestExtension.ensureStarted(QuarkusTestExtension.java:592)
        at io.quarkus.test.junit.QuarkusTestExtension.beforeAll(QuarkusTestExtension.java:640)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$12(ClassBasedTestDescriptor.java:395)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:395)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:211)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:84)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
        ... 36 more
Caused by: java.lang.NoClassDefFoundError: javax/enterprise/util/TypeLiteral
        at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3429)
        at java.base/java.lang.Class.getDeclaredConstructors(Class.java:2608)
        at io.quarkus.deployment.ExtensionLoader.loadStepsFromClass(ExtensionLoader.java:667)
        at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:217)
        ... 49 more
Caused by: java.lang.ClassNotFoundException: javax.enterprise.util.TypeLiteral
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:516)
        at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:466)
        ... 54 more
```

--- a/.mvn/maven.config
+++ b/.mvn/maven.config
@@ -1,3 +1,3 @@
--Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+-Dmaven.wagon.httpconnectionManager.ttlSeconds=60
 -Dmaven.wagon.http.retryHandler.requestSentEnabled=true
 -Dmaven.wagon.http.retryHandler.count=10

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -71,7 +71,7 @@
         <module>box</module>
         <module>braintree</module>
         <module>caffeine</module>
-        <!--<module>cassandraql</module> https://github.com/apache/camel-quarkus/issues/4488 -->
+        <module>cassandraql</module>
         <module>cbor</module>
         <module>compression-grouped</module>
         <module>consul</module>

--- a/pom.xml
+++ b/pom.xml
@@ -45,7 +45,7 @@
         <camel.docs.components.xref>${camel.docs.components.version}@components</camel.docs.components.xref><!-- the version in Camel's docs/components/antora.yml -->
         <camel.docs.branch>camel-${camel.major.minor}.x</camel.docs.branch><!-- The stable camel branch on which our Antora docs depends -->
         <camel.sb.docs.branch>camel-spring-boot-${camel.major.minor}.x</camel.sb.docs.branch><!-- The stable camel-spring-boot branch on which our Antora docs depends -->
-        <cassandra-quarkus.version>1.1.3</cassandra-quarkus.version><!-- https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
+        <cassandra-quarkus.version>1.2.0-alpha1</cassandra-quarkus.version><!-- https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
         <debezium.version>2.1.2.Final</debezium.version><!-- May go back to Camel's ${debezium-version} when they are in sync https://repo1.maven.org/maven2/io/debezium/debezium-bom/ -->
         <optaplanner.version>8.31.0.Final</optaplanner.version><!-- May go back to Camel's ${optaplanner-version} when they are in sync https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
         <quarkiverse-amazonservices.version>2.0.0.Alpha1</quarkiverse-amazonservices.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/amazonservices/quarkus-amazon-services-parent/ -->

--- a/poms/bom/src/main/generated/flattened-full-pom.xml
+++ b/poms/bom/src/main/generated/flattened-full-pom.xml
@@ -20010,9 +20010,9 @@
         <version>12.15.2</version><!-- com.azure:azure-sdk-bom:1.2.9 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>java-driver-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>4.15.0</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>java-driver-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>4.15.0</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
         <exclusions>
           <exclusion>
             <groupId>com.github.stephenc.jcip</groupId>
@@ -20023,70 +20023,70 @@
             <artifactId>jsr305</artifactId>
           </exclusion>
           <exclusion>
-            <groupId>io.dropwizard.metrics</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>metrics-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>io.dropwizard.metrics</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>metrics-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
           <exclusion>
-            <groupId>org.hdrhistogram</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>HdrHistogram</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>org.hdrhistogram</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>HdrHistogram</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>org.apache.tinkerpop</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>gremlin-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>3.5.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>org.apache.tinkerpop</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>gremlin-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>3.5.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
         <exclusions>
           <exclusion>
-            <groupId>org.yaml</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>snakeyaml</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>org.yaml</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>snakeyaml</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
           <exclusion>
-            <groupId>com.carrotsearch</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>hppc</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>com.carrotsearch</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>hppc</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
           <exclusion>
-            <groupId>com.jcabi</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>*</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>com.jcabi</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>*</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
           <exclusion>
-            <groupId>net.objecthunter</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>exp4j</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>net.objecthunter</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>exp4j</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
         </exclusions>
-        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>org.apache.tinkerpop</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>tinkergraph-gremlin</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>3.5.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>org.apache.tinkerpop</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>tinkergraph-gremlin</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>3.5.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.esri.geometry</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>esri-geometry-api</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.2.1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.esri.geometry</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>esri-geometry-api</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <optional>true</optional><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-client</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-client</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-client-deployment</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-client-deployment</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-test-framework</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-test-framework</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-mapper-processor</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-mapper-processor</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
         <groupId>com.datastax.oss</groupId><!-- com.datastax.oss:java-driver-bom:4.15.0 -->

--- a/poms/bom/src/main/generated/flattened-reduced-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-pom.xml
@@ -7313,12 +7313,12 @@
       <dependency>
         <groupId>com.datastax.oss.quarkus</groupId>
         <artifactId>cassandra-quarkus-client</artifactId>
-        <version>1.1.3</version>
+        <version>1.2.0-alpha1</version>
       </dependency>
       <dependency>
         <groupId>com.datastax.oss.quarkus</groupId>
         <artifactId>cassandra-quarkus-client-deployment</artifactId>
-        <version>1.1.3</version>
+        <version>1.2.0-alpha1</version>
       </dependency>
       <dependency>
         <groupId>com.datastax.oss</groupId>

--- a/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
@@ -7288,9 +7288,9 @@
         <version>12.15.2</version><!-- com.azure:azure-sdk-bom:1.2.9 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>java-driver-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>4.15.0</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>java-driver-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>4.15.0</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
         <exclusions>
           <exclusion>
             <groupId>com.github.stephenc.jcip</groupId>
@@ -7301,24 +7301,24 @@
             <artifactId>jsr305</artifactId>
           </exclusion>
           <exclusion>
-            <groupId>io.dropwizard.metrics</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>metrics-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>io.dropwizard.metrics</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>metrics-core</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
           <exclusion>
-            <groupId>org.hdrhistogram</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-            <artifactId>HdrHistogram</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+            <groupId>org.hdrhistogram</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+            <artifactId>HdrHistogram</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
           </exclusion>
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-client</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-client</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
-        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <artifactId>cassandra-quarkus-client-deployment</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
-        <version>1.1.3</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.1.3 -->
+        <groupId>com.datastax.oss.quarkus</groupId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <artifactId>cassandra-quarkus-client-deployment</artifactId><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
+        <version>1.2.0-alpha1</version><!-- com.datastax.oss.quarkus:cassandra-quarkus-bom:1.2.0-alpha1 -->
       </dependency>
       <dependency>
         <groupId>com.datastax.oss</groupId><!-- com.datastax.oss:java-driver-bom:4.15.0 -->

--- a/tooling/scripts/test-categories.yaml
+++ b/tooling/scripts/test-categories.yaml
@@ -61,6 +61,7 @@ group-03:
 group-04:
   - arangodb
   - debezium
+  - cassandraql
   - couchdb
   - influxdb
   - jdbc

